tests/Photons.test.ts

```diff
@@ -34,23 +34,23 @@ describe('Photons', () => {
     const photons = Photons.emptySpace(3, 5).addPhotonFromIndicator(0, 2, '>', 'V')
 
     expect(photons.ketString()).toBe('(1.00 +0.00i) |0,2,>,V⟩')
-    photons.propagatePhotons()
+    photons.propagatePhotons().actOnSinglePhotons()
     expect(photons.ketString()).toBe('(1.00 +0.00i) |1,2,>,V⟩')
-    photons.propagatePhotons()
+    photons.propagatePhotons().actOnSinglePhotons()
     expect(photons.ketString()).toBe('(1.00 +0.00i) |2,2,>,V⟩')
-    photons.propagatePhotons()
+    photons.propagatePhotons().actOnSinglePhotons()
     expect(photons.ketString()).toBe('')
   })
```

Comment:
It is a test for propagation, here we *don't* want to use `.actOnSinglePhotons`.

Reply:
In that case, a separate test that does both on empty board is needed. I can create one if you want.